### DISCUSSION TITLE
attribute "thumbnailphoto"

### DISCUSSION THREAD

**Discussion by shaedrich** [DirectoryTree/LdapRecord#358] [Q&A]
Hey, is it just me or is the attribute `thumbnailphoto` missing and cannot be retrieved (anymore)?

**Comment by stevebauman** [comment]
Hi @shaedrich, nothing has changed in regard to attribute retrieval. I would dump the model's attributes to ensure it exists and contains the proper data 👍

**Reply by shaedrich** [reply]
That's what I did and the property was missing.

**Reply by stevebauman** [reply]
"Was" missing, or "is" missing?

If it was previously there, I would check the accounts permissions that you're using to bind to your directory with, or check to make sure your application hasn't been modified accidentally to omit this attribute from being returned.

Without extensive code and examples detailing the issue, I can't help any further than that.

**Reply by shaedrich** [reply]
It was missing when I last checked (ereyesterday) and is still missing. My application hasn't been modified in terms of the LDAP package. Thank you for your help anyway.
